tox.ini

```diff
@@ -38,6 +38,15 @@ commands =
     black plugins/ --check
     black examples/ --check
 
+[testenv:black-diff]
+basepython = python3.10
+deps =
+    black==23.3.0
+commands =
+    black composio/ scripts/ tests/ --diff
+    black plugins/ --diff
+    black examples/ --diff
```

Comment:
Why do we need diff? Black is a code formatter it's not going to change you code, you can just run black as formatter without worrying about code breaking

Reply:
When Locally tesing the code formatting, then black-diff is useful I felt. Was for me, to see what I misformatted before fixing it. But , If you prefer I can remove it. @angrybayblade